src/backend/joanie/tests/base.py

```diff
@@ -61,3 +61,83 @@ def generate_token_from_user(user, expires_at=None):
             token, the jwt token generated as it should
         """
         return generate_jwt_token_from_user(user, expires_at)
+
+
+class BaseLogTestCase(TestCase):
```

Comment:
Why not making a Mixin rather than extending `TestCase` ? What if I want to use this assertion but I also want to use an other test case ?